nnstreamer_example/custom_example_LSTM/dummy_LSTM.c

```diff
@@ -70,6 +70,22 @@ pt_exit (void *private_data, const GstTensorFilterProperties * prop)
   free (data);
 }
 
+/**
+ * @brief copy tensor info
+ */
+#define copy_tensor_info(_info, _data) \
+  do { \
+    unsigned int i = 0; \
+    unsigned int num_tensors = 0; \
+    assert (_info != NULL); \
+    assert (_data != NULL); \
```

Comment:
I think such `assert ()` seem to be redundant.

Reply:
agree. will update this.

Reply:
done.